app/controllers/manage_collections_controller.rb

```diff
@@ -1,12 +1,11 @@
 # :nodoc:
 class ManageCollectionsController < PagesController
   include BulkUpdates
-  include ManageCollectionsHelper
 
-  layout 'manage_collections'
+  layout 'manage_metadata'
```

Comment:
I thought we had discussed keeping a bare `ManageMetadataController` that the controllers for ManageCollections, ManageVariables, and ManageServices could all inherit from. Then we wouldn't need to have to declare this layout in the three different controllers. Had that decision been changed, or did those edits get lost somehow? I recall seeing that and discussing it on the PR review for 1033 the other day.